documentation/views/control.rst

```diff
@@ -6,57 +6,15 @@ Flexibility opportunities
 
 Flexibility opportunities have commercial value that users can valorise on.
 When FlexMeasures has identified commercial value for flexibility opportunities, the user is suggested to act on them.
-This might happen manually (operators seeing opportunities in the FlexMeasures UI) or in an automated fashion
-(scripts reading out suggested flexible schedules from the FlexMeasures API and implementing them to local operations if possible).
+This might happen in an automated fashion (scripts reading out suggested flexible schedules from the FlexMeasures API and implementing them to local operations if possible) or manually (operators seeing opportunities in the FlexMeasures UI).
```

Comment:
flexible schedules -> schedules
operators seeing opportunities in the FlexMeasures UI -> operators agreeing with the opportunities identified by FlexMeasures and acting on the schedules recommended